src/TLuaInterpreter.cpp

```diff
@@ -17197,6 +17198,108 @@ int TLuaInterpreter::getMouseEvents(lua_State * L)
     return 1;
 }
 
+int TLuaInterpreter::setConfig(lua_State * L)
+{
+    auto& host = getHostFromLua(L);
+    QString key = getVerifiedString(L, __func__, 1, "key");
+    if (key.isEmpty()) {
+        return warnArgumentValue(L, __func__, "you must provide key");
+    }
+
+    auto success = [&]()
+    {
+        if (mudlet::debugMode) {
+            TDebug(Qt::white, Qt::blue) << qsl("setConfig: a script has changed %1\n").arg(key) >> &host;
+        }
+        lua_pushboolean(L, true);
+        return 1;
+    };
+
+    if (host.mpMap && host.mpMap->mpMapper) {
```

Comment:
haven't solved this nicely, but the fact that these options only work when the mapper is enabled means that the warning message down below will falsely say Mudlet has no idea what these mapper options are.

Reply:
That doesn't seem to address the problem at hand - try setting mapper-related preferences while it is closed to see what I mean.